.github/size-plugin.yml

```diff
@@ -1,2 +0,0 @@
-size-files:
-- size-plugin.json
```

Comment:
AFAIK this is unused? I think it comes from back when we were using preact-cli? Not quite sure tbh.